content/changelog-stable/index.html.haml

```diff
@@ -5,38 +5,19 @@ show_ratings: true
 has_rss: true
 ---
 
-%div{:class => 'app-banner'}
-  See the
-  %a{:href => '/doc/upgrade-guide/'}
-    LTS upgrade guide
-  for advice on upgrading Jenkins.
-
-%div{:class => 'app-banner'}
-  See the
-  %a{:href => '/download/lts'}
-    LTS Release Line
-  for more information on Jenkins LTS.
-
 .ratings
   - # source: https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/lts.yml
   - site.changelogs[:lts].reverse_each do | release |
     - if Gem::Version.new(release.version) <= Gem::Version.new(site.jenkins.stable)
-      = partial('release-header.html.haml', :release => release)
-      %p
-        %small
-          %a{:href => "/changelog-stable/#{release.version}/" }
-            permalink to this entry
-      -if release.changes and release.lts_changes and release.lts_baseline
-        %div
-          %strong
-            -# TODO figure out how to get rid of this
-            = "Changes since #{release.lts_baseline}:"
-      %ul.image
-        = partial('changelog-changes.html.haml', :changes => release.changes)
-      -if release.changes and release.lts_changes
-        %div
-          %strong
-            = "Notable changes since #{release.lts_predecessor}:"
-        %ul.image
-          = partial('changelog-changes.html.haml', :changes => release.lts_changes)
+      %div
```

Comment:
If you go back to `2.19.4` its still using the old design. FWIW I think we could archive going back a certain amount of time we don't need to keep growing this page infinitely

<img width="1027" alt="image" src="https://github.com/user-attachments/assets/98db8b8c-62f8-4533-80ba-fe1c0a1aa32b">

Reply:
> its still using the old design

That's from the HTML archive, when we didn't use YAML yet: https://github.com/jenkins-infra/jenkins.io/blob/94c5f8cd06c6392c5238f0bdd46d66111c515f45/content/_data/changelogs/lts.yml#L5

I think this difference is acceptable and we don't need to rewrite ancient HTML for the new format.

Reply:
I don't think we should delete changelogs, but if we do, we should start with historical weekly changelogs.

Reply:
Makes sense, I read your suggestion as completely removing. I'd keep at least 2 years of changelogs around though for LTS.

Reply:
Created an issue to track: https://github.com/jenkins-infra/jenkins.io/issues/7502